### PR TITLE
Seek-A-Design About Me Page HTML and CSS

### PR DESCRIPTION
**Pull request by vonschlichten**
Set up the basic skeleton/structure of the HTML and implemented some basic styling to the elements.

### CODE REVIEW

**Comment by anthonypecchillo** [comment]
@vonschlichten - Please:

- Refactor your Pull Request Title to indicate what this PR is about. Maybe something like, "Seek-A-Design Home Page"
- Add a description that briefly lists the what this PR contributes to the work mentioned in the Pull Request Title. (You can practice using Markdown here!)

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: **Lines 13-21:**

In our designer's mockups, it appears that the text, "CUSTOM DESIGNS - MARKETING - ..." is grouped together with the navbar. 

Perhaps these two DOM elements should be contained within the same `<div>`? This would allow use to position them on the screen together when it comes time to worry about the page's layout.

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: **Lines 19-21:**

`<span>` tags are generally used to format **sub-sections** of text in-line, kind of like this:

```
<p>I am a paragraph with lots of sentences. Every once in a while, <span style="color: green;">some of my text is green</span>, but usually, it is just black. Sometimes I get a little wild and <span style="color: pink;">turn my text pink</span>, but I <span style="color: green;">rarely</span> do that.</p>
```

But we don't often use them to wrap an entire body of text. Instead, just go with a `<div>` tag.

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: To better match the designer's mockups:

- Try adding a `<hr>` tag below `<div class="whois">` to get a "horizontal rule" (a horizontal line).
- In `style.css`, select all of your `<hr>` tags by writing `hr { /* style code goes here */ }` and add some CSS to format the line.
- Back in `index.html`, don't forget to add a TODO comment above the `<hr>` tag to remind yourself to come back later and replace this line with the actual static asset our designer forgot to provide us with!

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: Let's try not to rely on `<br/>` tags to create horizontal spacing between elements. It's better if we begin building the habit of using CSS for this.

This is what's known as keeping a **"separation of concerns,"** and will become more important as our code grows. 

- We'll let our HTML be the nouns; "What is in our app?"
- We'll let our CSS be the adjective; "What are our nouns (the HTML) like?" 
- We'll let our Javascript be the verbs; "What do our nouns (the HTML) do?"

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: Same as Line 24.

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: To better match the mockups:

- Consider making this two separate `<h2>` tags, so we can vertically stack our text.

To improve our code style:

- Kill the empty space between the opening tag and the text "WHO".

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: Nice work. Now, can you:

- [Easy] - Add the remaining two paragraphs? (Make them each a separate `<p>` tag.
- [Harder] - Make it so that if the total text in this section doesn't fit inside its container, the text becomes scrollable.
    - Do this **without** allowing the title to scroll!

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: Similar to Line 25, except a bit harder...

Here, you can imagine that we have some sort of **persistent data store**; a place where we can store data long term. What data? Individual reviews that have been left by previous clients!

How many reviews do we have? Who knows. They will accumulate over time! When we get to the javascript part of this page, we'll have to make the length of this list **dynamic**; it will change based on the length of the array that contains all of the reviews.

For now, we can hard-code each review, but we should **code them as an un-ordered list of paragraphs**, rather than just a series of paragraphs.

To do this:

- Use the `<ul>` tag to create an unordered list.
- Use a bunch of `<li></li>` tags, nested inside the `<ul></ul>`.
- Inside each `<li>` tag, create a paragraph (`<p></p>`) containing the text for a single review.
- Go to your `style.css` file, and figure out how to remove the bullet points from your list.
- Continue styling the list to match the styling of the mockup.

File: dist/style.css
Comment (anthonypecchillo) [inline dist/style.css]: This is really cool! It has nothing to do with assignment, but that doesn't stop it from being fucking awesome.

Comment (anthonypecchillo) [reply dist/style.css]: I also suspect that these styles mmaayyyy be copy-paste code. Either that, or you probably don't need my advice on CSS from here forward. ¯\\_(ツ)_/¯

File: dist/style.css
Comment (anthonypecchillo) [inline dist/style.css]: Cool background images, bruh.

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: **Lines 22-31:**

We should probably wrap this whole section in a parent `<div>`. I recommend this because these two boxes are meant to be two columns in the same box, as per the mockup.

**Review by anthonypecchillo — CHANGES_REQUESTED** [review]
## First Review: HTML

### General Feedback:

Nice work so far John! This is a good start. It looks like you had a lot of fun with this too. I really like the glowing rainbow hover effects, and the background images for the two content sections of the site.

I gave you a light review of your HTML where it seemed appropriate. It's hard to get too much deeper into this because it seems that you purposefully disregarded most of the provided mockup, so most things I'd point out as needing improvement were probably intentional.

Please do the following before continuing so that you don't lose the fun work you've completed:

1. Open your terminal
2. Navigate to the directory where you were working on this project
3. Type `git checkout -b fun-version`  (This should create a new Git Branch, so that you can have two versions of your work.)
4. Type `git branch` to verify that you now have two separate branches.
5. Type `git checkout master`
6. Type `git branch` again to verify that you have moved from the `fun-version` branch to your `master` branch.
7. Now, in your master branch, try to implement the actual mockup. 🌝

### Next Steps:

- Let's continue building out the HTML Skeleton, without focusing too much on the CSS for now. (Do some styling, but only what is essential.) This shouldn't take very long to get right, and once we have a fully-built HTML skeleton we can merge your code into version control on GitHub, and you'll have your first successful Pull Request and contribution to a professional project!
- Then, we'll get started on a second Pull Request where we'll focus entirely on the styling of your HTML using CSS. This might take a bit longer, but will be well worth the ride.
- Once your second PR is complete, approved, and merged, we can start building out the Javascript/jQuery to add some functionality to the page (the verbs), and also do a little refactoring of the HTML/CSS.

Onward and upward 👍

File: dist/index.html
Comment (vonschlichten) [reply dist/index.html]: Alright. I have combined them into the same div and now they are inline. I will work on figuring out how to have the text below the navbar.

File: dist/style.css
Comment (vonschlichten) [reply dist/style.css]: you already know my g ... ya tu sabes

File: dist/index.html
Comment (vonschlichten) [reply dist/index.html]: I was able to nest the text "CUSTOM DESIGNS - MARKETING - ..." within the navbar div. I believe changing parent container from inline, to block enabled me to position the  text below the navbar.

File: dist/index.html
Comment (vonschlichten) [reply dist/index.html]: Gotcha!

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: **Lines 14-24:**

There are many ways to accomplish this goal, and your way is one of them! 

However, *ideally*, a nav section of HTML will be nested inside a `<nav></nav>` tag. Then, each menu item will be a list item (`<li>`) in an unordered list (`<ul>`). Like this:

```
    <nav>
      <ul class="navbar">
        <li>
          HOME
        </li>
        <li>
          PROJECTS
        </li>
        ...
      </ul>
    </nav>
```

Our HTML is simple enough to make this refactor now, so let's make it happen! 

Removing the bullet points from the list might take a little CSS magic, but it looks like you've figured out that technique already in your `.seeking` class, so I'm confident you can accomplish this here as well. 👍

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html:22]: **Lines 15, 18, 21, and 24:**

Careful! You're over-indenting by two spaces here.

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: *Lines 34, 38, and 42:*

Can we wrap our text here, keeping it less than 80 characters wide? 
(Use the vertical line in your editor to easily identify the 80 character wide mark.)

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html:83]: Great in this div! Love how you handled the list of reviews here! 💯

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: Love how you chose to wrap the text here! ✅

*Can we just update it a bit to make it less than 80 characters wide on each line?*

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html:129]: Nice, love that you left room for this section. Let's fill this in with some HTML for each icon. 

Fortunately, we have a team member who's already built a system of 3 icons, each on a solid circular background. Take a look at the code in the `landon` branch to help you get started with this section. 

*Note:* There is no shame is collaborating with your team mates. In fact, it's better to re-use already existing code than it is to re-write it! The CSS styling may be a bit different here (because the icons are smaller), but the HTML structure will likely be quite similar. 👍

**Review by anthonypecchillo — CHANGES_REQUESTED** [review]
# Second Review: HTML

### General Feedback:

Nice work John, now we're getting somewhere! 🚀

Nice work satisfying my revision requests. Now I've got a few more for you. (See comments and additional challenge below.)

##### Additional Challenge:

In addition to the revision requests below, I'd like you to consider the following with respect to our designer's mockup:

- The mockup has the two "Who Is Seek-A-Design" section and the "Seeking-A-Review" section as two columns _in the same box_.
- You currently have these two columns as separate boxes, contained by nothing but the body of the page.

**Challenge:** Can you refactor your code to make your two boxes separate boxes turn into two columns within the same box?

### Next Steps:

Let's keep working on getting the HTML skeleton solid, without worrying too much about CSS updates, unless you feel they are absolutely necessary to make the HTML make sense. This is our focus for now so that we can land your first PR, and move on to a second more CSS focused PR.

**Let's get the entire HTML and CSS wrapped up (PR 1 and 2) by Wednesday Morning. Remember, Code Review takes time, so get your PR submissions in as early as possible.**

Then, we can spend the remainder of the week working on jQuery.

File: dist/index.html
Comment (vonschlichten) [reply dist/index.html]: Alright, I went ahead and reformatted this section of my code as directed. I removed the bullet points in an earlier task previously, so I was able to just copy and paste that css code. ```After changing <div> and <button> to <nav> <ul> <li>``` elements,  will I still be able to apply the same css that I used on my buttons to my li's? 

I went to Landons branch to check out his html/css so I could try transferring over the button css styles over. Is there a way to view the files (as a webpage that I can inspect) on github, or do I need to download them and open them manually? Any feedback is appreciated!

File: dist/index.html
Comment (vonschlichten) [reply dist/index.html]: I have  the 80 character line in atom now. Is there any sort of wrapping package available that will drop the text to the next line one it reaches the 80th character, that you know of?

File: dist/index.html
Comment (vonschlichten) [reply dist/index.html:129]: Okay, so I came close but I'm still having difficulties arranging/aligning with grid. I'll spend some more time reviewing some tutorials tomorrow.

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: Okay nice, good to include this, but let's move it up to line 5 and keep it with the other `meta` tags to keep things organized.

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: I like the idea here (giving this particular `li` tag a unique ID, `"left-button"`, but I don't like the ID's name because:

1. It's not actually a `button` tag.
2. What if in the future there are other `button` tags on the page? This may get confusing for the reader.

Switch this to something like `left-nav-item` to increase the readability of your code, and don't forget to update the selector in your `style.css` from `#left-button` to `#left-nav-item` as well.

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: Did you mean to name the class `nav-contact` here?

Comment (anthonypecchillo) [reply dist/index.html]: Also, shouldn't this `li` get a `id="right-button"`? (Or similar, see comment about `#left-button` above.

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html:19]: Great work on the navbar's HTML update. You nailed it with my revision request. 👍

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: Since we know this is going to get updated in the future, I would put a comment above the `<hr>` that says:

```
<!-- TODO: Replace with actual horizontal rule static asset when designer supplies the file. -->
```

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html:109]: Capitalize the 'A' in 'Seek-a-Design'.

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: Hm...from a UX (User Experience) perspective, I think it would be great if the entire circle + icon here was clickable as a link. Right now, you have to specifically click the icon. This might be only a slight annoyance with a mouse, but on a touch screen could become agonizing.

Have any ideas on how we can fix this?

File: dist/index.html
Comment (anthonypecchillo) [inline dist/index.html]: Also, I believe the docs for Font Awesome say that when using their icons, we should use either:

- A `<span>` tag
- An `<i>` tag

Check out the [Official Font Awesome Documentation](https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use) to confirm that I am right, and see example code.

File: dist/index.html
Comment (anthonypecchillo) [reply dist/index.html:129]: Sounds good - if you'd like some help with CSS Grid, we can hop on a call. 

However, for this PR, we are exclusively focused on getting the HTML right in order to land the PR. As long as we get the HTML finished, we can move on to the next PR.

(Land = Code Reviewers accept your work, and merge your code into the code base.)

File: dist/index.html
Comment (anthonypecchillo) [reply dist/index.html]: *Hint:* You can nest HTML inside an `<a></a>`. 😉

File: src/index.js
Comment (anthonypecchillo) [inline src/index.js]: Hm, this doesn't appear to be working. I bet that `www.google.com` needs to be a string. Wrap it in quotes and see if it works!

**Review by anthonypecchillo — CHANGES_REQUESTED** [review]
## Second Review: HTML

### General Feedback:

Good stuff @vonschlichten! We're really getting close to wrapping this Pull Request up and getting your first contribution to a real-world project merged into a real-world codebase!

I particularly like how:

- You nailed it with the NavBar.
- You were able to get the two separate boxes in the body into a single section with columns.
- The inclusion of the social icons at the bottom of the page, in their own section.

In the _Next Steps_ section of my previous review, I wrote:

> Let's continue building out the HTML Skeleton, without focusing too much on the CSS for now. (Do some styling, but only what is essential.) [...] once we have a fully-built HTML skeleton we can merge your code into version control on GitHub, and you'll have your first successful Pull Request and contribution to a professional project!

For this reason, I only dug into your HTML for now. However, I see the results of your CSS changes in my browser, and things are looking better! I'll dig into CSS in your next PR.

### Next Steps:

- Let's wrap up the HTML and this PR in one final revision. The requests are short and simple, so let's knock thing out in the next 24 hours!
- Then, we'll get started on a second Pull Request and focus exclusively on the styling. I'll give you some help with CSS Grid once you're here.
- Once your second PR is complete, approved, and merged, it's Javascript time 😎

File: dist/index.html
Comment (anthonypecchillo) [reply dist/index.html]: **Forgot to add this comment while in Code Review mode so adding it as a thread comment here:**

Please include the Seek-A-Design logo on the page. Some people like to use `` tags for this, but personally, I like to use a `<div>` and set the background property with CSS, so a `div` with CSS styles will be fine. The CSS might look like this:

```
#logo {
  background-image: url('A string with the **relative path** to the image in your file system');
  background-repeat: no-repeat;
  background-size: cover;
  height: 100%;   // Makes a GIANT assumption that the containing div is a square
  width: 100%;    // Makes a GIANT assumption that the containing div is a square
}
```

Comment (vonschlichten) [reply dist/index.html]: I am having difficulty getting the image to appear using CSS, specifically using the **relative** path here. I'm researching now but still cannot get the image to appear.

File: dist/index.html
Comment (vonschlichten) [reply dist/index.html]: Alright, so I tried nesting my html within a tags, and I was able to get the circle to be clickable -- but then the logos got all fucked up. I'll keep at it for sure, but in what direction I couldn't tell you.

File: dist/index.html
Comment (vonschlichten) [reply dist/index.html]: This is code that I borrowed from Landons html/css since he already created the buttons. You are correct, and I used the i tags for my nav bar links.

File: src/index.js
Comment (vonschlichten) [reply src/index.js]: I don't even know how this got here? I haven't tried implementing any javascript yet.

File: dist/index.html
Comment (vonschlichten) [reply dist/index.html]: Update: 
```
        <a href="https://www.facebook.com/SeekADesign/">
          <div class="circle-icon-green">
            <i class="fab fa-facebook-f"></i>
          </div>
        </a>

        <a href="https://www.twitter.com/SeekADesign/">
          <div class="circle-icon-purple">
            <i class="fab fa-twitter"></i>
          </div>
        </a>

        <a href="https://www.instagram.com/seekadesign/">
          <div class="circle-icon-green">
            <i class="fab fa-instagram"></i>
          </div>
        </a>

What do you think about this method of using the anchor tag?

File: dist/index.html
Comment (vonschlichten) [reply dist/index.html]: Update - I was able to get the image to appear but I moved the image file location where the css file is located. I know that is kind of cheating but I was having difficulty with the relative path. example ../ ../

So, the Seek-A-Design logo in on the page now. I know we are focusing on the HTML portion and I am trying to stick to that. I suppose we can review the styling and alignment in the next review.

File: dist/index.html
Comment (vonschlichten) [reply dist/index.html]: Comment left, and I even attempted implementing this image using the css on my hr but it isnt' working yet : /

File: dist/index.html
Comment (vonschlichten) [reply dist/index.html]: Corrected.

File: src/index.js
Comment (vonschlichten) [reply src/index.js]: Wrapping in quotes did not make it work!

File: dist/index.html
Comment (anthonypecchillo) [reply dist/index.html]: This post is making me realize that we need to have a group call on GitHub, group project workflow, etc, to better enable the three of us to work as a team. Sorry if this has been difficult so far.

As far as the styling goes, for the most part, the CSS should be transferable, with only minor exceptions.

File: dist/index.html
Comment (anthonypecchillo) [reply dist/index.html]: I'm sure there is, but it is best to just hit enter and do it manually yourself, otherwise there could be errors. Let's just stick with that for now. I have had bad experience with text-wrap in editors in the past.

File: dist/index.html
Comment (anthonypecchillo) [reply dist/index.html]: No worries - hit up on Slack when you need help with a particular line of code like this! For now, here is the relative path for the image, with respect to the CSS file:

`./assets/logos/SeekADesign_VectorLogo.svg`

Try putting that in the CSS snippet above. If it works, great. If not, shoot me a Slack message and we'll debug it together!

File: dist/index.html
Comment (anthonypecchillo) [reply dist/index.html]: Ah don't worry about doing that! That's our designer's job. It's on her to provide us with an SVG of this image. We just need to stick it in an `` tag.

File: dist/index.html
Comment (anthonypecchillo) [reply dist/index.html]: Ah, I can see how that would happen. But that's okay, its just going to take a bit of CSS work to get everything aligned right.

Let's leave it for now, and worry about that in the CSS PR. I'm happy with this for now, and if we have to refactor in the next PR, so be it. Good work.

File: src/index.js
Comment (anthonypecchillo) [reply src/index.js]: Hm, well in this case, fuck it!

Comment (anthonypecchillo) [reply src/index.js]: We'll delete it later lol. Leave it for now.

File: dist/style.css
Comment (anthonypecchillo) [inline dist/style.css]: Put a space between `display` and `block;`

I need to get you guys a linter to point out syntax errors. Coming soon.

File: dist/style.css
Comment (anthonypecchillo) [inline dist/style.css]: **For relative paths:**

`./` means "starting at the current directory"
`../` means "go backwards one directory"
`../../` means "go backwards two directories"

Please put this file back in the `./dist/assets/other` directory, and change the path here to:

`./dist/assets/other/OrnamentalDividier.svg`

File: dist/style.css
Comment (anthonypecchillo) [inline dist/style.css]: **For relative paths:**

`./` means "starting at the current directory"
`../` means "go backwards one directory"
`../../` means "go backwards two directories"

Please put this file back in the `./dist/assets/logos` directory, and change the path here to:

`./dist/assets/logos/SeekADesign_VectorLogo.svg`

**Review by anthonypecchillo — CHANGES_REQUESTED** [review]
## Third Review: Nitty Gritty Details

### General Feedback:

Great work. Just a few minor revisions, and we're good to merge this thing in. Let's get it merged in this morning. I'll be online.

### Next Steps:

- Make the requested revisions and let's land this thing!
- Start a new branch in your repo to start working on your next PR by following the instructions I've provided in the README.md file.

File: dist/index.html
Comment (vonschlichten) [reply dist/index.html]: Wait, I thought she did provide us with ornamentaldivider.svg? So, we couldn't go about displaying this image the same way we did the logo? gotcha!

File: dist/index.html
Comment (vonschlichten) [reply dist/index.html]: I was able to get entire circle and icon clickable!

File: src/index.js
Comment (vonschlichten) [reply src/index.js]: fershizzle

**Comment by anthonypecchillo** [comment]
Looks good to me! 👍

**Pull request merged**
